src/platform/testing/e2e/nightwatch-assertions/isAllyFocusableCount.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Checks if the count of focusable elements is correct. Focusable elements are those
+ * in the normal tab order (native focusable elements or those with tabIndex <= 0).
+ *
+ * ```javascript
+ *  this.demoTest = function (client) {
+ *    client.assert.isAllyFocusableCount("body", 10, "Your custom message");
+ *  };
+ * ```
+ *
+ * @method isAllyFocusableCount
+ * @param {string} [selector] The selector (CSS / Xpath) used to locate the element
+ * @param {number} [count] The number of focusable elements on the page
+ * @param {string} [message] Optional log message displayed in output. Defaults to this.message.
+ * @api assertions
+ */
+exports.assertion = function isAllyFocusableCount(selector, count, message) {
```

Comment:
What do you think of calling this something like `hasFocusableCount`? It's really asserting that it _has_ a certain number of focusable elements, not that it _is_ a focusable count, y'know?